### PR TITLE
Basic functionality

### PR DESCRIPTION
**Pull request by jklenzing**
This PR adds setup files, community docs, and travis/coveralls infrastructure.  The module currently acts as an instrument library that can be loaded into pysat.

Old Method:
```
import pysat
sgp4 = pysat.Instrument(platform='pysat', name='sgp4')
```

New Method (after install of pysatMissionPlanning):
```
import pysat
from pysatMissionPlanning.instruments import pysat_sgp4

sgp4 = pysat.Instrument(module=pysat_sgp4)
```

Once pysat/pysat#291 is merged, sgp4 can be permanently added to a user's library provided they have installed both packages.

### CODE REVIEW

**Comment by rstoneback** [comment]
Thanks Jeff.

clone and install works. I can import the package and make a pysat.Instrument.

Typo in the commands,
import pysat
from pysatMissionPlanning.instruments import pysat_sgp4

sgp4 = pysat.Instrument(inst_module=pysat_sgp4)

print(sgp4)
pysat Instrument object
-----------------------
Platform: pysat
Name: sgp4
Tag: 
Satellite id: 

Data Processing
---------------
Cleaning Level: clean
Data Padding: None
Keyword Arguments Passed to load(): {}
Custom Functions : 
    <function add_quasi_dipole_coordinates at 0x123fa7b90>
    <function add_aacgm_coordinates at 0x123fa7c08>
    <function calculate_ecef_velocity at 0x123fa7b18>
    <function add_sc_attitude_vectors at 0x123fa7aa0>
    <function add_iri_thermal_plasma at 0x123fa7c80>
    <function add_hwm_winds_and_ecef_vectors at 0x123fa7cf8>
    <function add_igrf at 0x123fa7d70>
    <function project_ecef_vector_onto_sc at 0x123fa7e60>
    <function project_hwm_onto_sc at 0x123fa7ed8>
    <function add_msis at 0x123fa7de8>

Orbit Settings
--------------
Orbit properties not set.

Local File Statistics
---------------------
Number of files: 366
Date Range: 01 December 2017 --- 01 December 2018

Loaded Data Statistics
----------------------
No loaded data.

**Comment by rstoneback** [comment]
Creating issue in pysat to provide user feedback when pysat registers a non-pysat keyword.

File: pysatMissionPlanning/tests/test_core.py [hunk 1]
```diff
@@ -15,6 +15,6 @@ def teardown(self):
     def test_module_loading(self):
         """Check if instrument module can be loaded as a pysat instrument"""
 
-        testInst = pysat.Instrument(module=self.module)
+        testInst = pysat.Instrument(inst_module=self.module)
```
Comment (rstoneback) [inline pysatMissionPlanning/tests/test_core.py]: You could also test that testInst.platform and testInst.name match the corresponding attributes of self.module . I think that would've at least caught this issue.

Could also test that the correct load, clean, etc. from self.module are also present as testInst._list_rtn etc

Comment (jklenzing) [reply pysatMissionPlanning/tests/test_core.py]: I noticed that the coverage went up a bit after fixing this.

Comment (rstoneback) [reply pysatMissionPlanning/tests/test_core.py]: Cool. I like my test suggestions, but those are probably better within pysat, and perhaps out of scope.

Comment (jklenzing) [reply pysatMissionPlanning/tests/test_core.py]: Ok. Will think about bout this as part of the next one.

**Comment by asher-pembroke** [comment]
Hi, I didn't have time to review this before it was merged. Let me know if you still want me to take a look

**Comment by rstoneback** [comment]
No worries, no need to work weekends. Jeff and I just got a bit ahead.

**Pull request merged**